web/styles/subscriptions.css

```diff
@@ -662,7 +662,7 @@ h4.user_group_setting_subsection_title {
 .user-groups-list,
 .streams-list {
     position: relative;
-    overflow: auto;
+    overflow: visible;
```

Comment:
I had a doubt if this could break anything, but seems fine in testing. But maybe we can remove the CSS here since `visible` is the default value.